### DISCUSSION TITLE
Chapterize-Audiobooks General Discussion

### DISCUSSION THREAD

**Discussion by patrickenfuego** [patrickenfuego/Chapterize-Audiobooks#12] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

 To get started, comment below with an introduction of yourself and tell us about what you do with this community. 

If you have a specific feature request after engaging in discussion, please open an Issue so it can be tracked in the Projects tab.

**Comment by ashleyconnor** [comment]
Just used this to chapterize an audiobook that was a single mp3 file. Works amazingly! Thank you!!!

**Comment by dmupp3t** [comment]
Currently using this to chapterize a 62 hour long audiobook. It's been running for a good clip now. On average how long does it normally run for? Just ball park.

**Reply by ashleyconnor** [reply]
It did a 13 chapter audiobook in around 20 minutes. I tried docker at first but couldn't get it to complete.

**Reply by dmupp3t** [reply]
It eventually finished after 8+ hours - This particular book has 147 chapters :)

**Reply by ashleyconnor** [reply]
The War and Peace effect :)
